example/src/main/java/com/posthog/java_example/App.java

```diff
@@ -0,0 +1,85 @@
+package com.posthog.java_example;
+
+import java.util.HashMap;
+import java.lang.Thread;
+
+import com.posthog.java.PostHog;
+
+// Testing against local PostHog snapshot jar:
+// 1. `mvn install` in posthog folder to put the jar in local repo (~/.m2/repository/com/posthog/java/posthog)
+// 2. Update the api key and host values to yours
+// 3. `mvn compile exec:java` in example folder
+public final class App {
+    private static final String POSTHOG_API_KEY = "phc_gFipiKER7U6JWYEHTk6C240reCnrWLBcKivOcK7JOMh";
+    private static final String POSTHOG_HOST = "http://188.166.138.169";
+
+    private App() {
+    }
+
+    public static void main(String[] args) {
+        PostHog posthog = new PostHog.Builder(POSTHOG_API_KEY).host(POSTHOG_HOST).build();
+
+        posthog.capture("distinct id 1", "movie played", new HashMap<String, Object>() {
+            {
+                put("movie_id", 123);
+                put("category", "romcom");
+            }
+        });
+
+        posthog.identify("distinct id 1", new HashMap<String, Object>() {
```

Comment:
This got me thinking about how our server libs are both confusing and also probably a key reason for some of the issues we had with merging people/duplicates.

We use `identify` to set user props (and also tag the person as identified) but we don't pass in an anonymous distinct ID. I believe the repercussions of that in the plugin server used to be problematic. 

While the method is effectively the same as `posthog-js`, ultimately it behaves quite differently in stateless libs.

Nothing actionable for you here, but I'm wondering what `identify` should really be in server side libs